### PR TITLE
hoverboard added (Day 36)

### LINKED ISSUE #74: Add Hover Board (Day 36)
**Reference:**
[link](https://50projects50days.com/projects/hoverboard/)

### PR DESCRIPTION
**Pull request by sank2000**
closes #74

### CODE REVIEW

File: src/pages/hoverboard.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    cardRef.current.style.boxShadow = `0 0 2px ${color}, 0 0 10px ${color}`;
+  }
+
+  function removeColor() {
+    cardRef.current.style.background = '#1d1d1d';
+    cardRef.current.style.boxShadow = '0 0 2px #000';
+  }
+
+  return <Card key={ind} ref={cardRef} onMouseOver={setColor} onMouseOut={removeColor} />;
+}
```
Comment (KrishnaMoorthy12) [inline src/pages/hoverboard.tsx:30]: Please keep main [default export] component at the top.

**Review by KrishnaMoorthy12 — CHANGES_REQUESTED** [review]
Works fine, but I request you to make the mentioned changes.

**Comment by KrishnaMoorthy12** [comment]
Also, there's this warning:
```
Warning: Each child in a list should have a unique "key" prop.

Check the top-level render call using <hoverboard__Main>. See https://reactjs.org/link/warning-keys for more information.
    at Box (webpack-internal:///./src/pages/hoverboard.tsx:24:3)
    at HoverBoard
    at App (webpack-internal:///./src/pages/_app.tsx:24:3)
    at AppContainer (/home/krish/github/7days-50projects/node_modules/next/dist/next-server/server/render.js:25:748)
```

File: src/pages/hoverboard.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    cardRef.current.style.background = color;
+    cardRef.current.style.boxShadow = `0 0 2px ${color}, 0 0 10px ${color}`;
+  }
+
+  function removeColor() {
+    cardRef.current.style.background = '#1d1d1d';
+    cardRef.current.style.boxShadow = '0 0 2px #000';
+  }
+
+  return <Card key={ind} ref={cardRef} onMouseOver={setColor} onMouseOut={removeColor} />;
```
Comment (PranavGPR) [inline src/pages/hoverboard.tsx:29]: Instead of using onMouseOver, use onMouseEnter

**Review by PranavGPR — CHANGES_REQUESTED** [review]
**Last 6 squares are out of that structure**

image

**Comment by KrishnaMoorthy12** [comment]
> **Last 6 squares are out of that structure**
> 
> image

@sank2000 opinion on this?

File: src/pages/hoverboard.tsx (hunk 2 above)
Comment (KrishnaMoorthy12) [reply src/pages/hoverboard.tsx]: also this @sank2000

**Review by KrishnaMoorthy12 — APPROVED** [review]
good job

**Review by PranavGPR — APPROVED** [review]
Nice work!

**Pull request merged**